tasks/sep/006-0-sc-guardian-updates/SignFromJson.s.sol

```diff
@@ -0,0 +1,528 @@
+    function superchainConfig() external view returns (address superchainConfig_);
+    function version() external view returns (string memory);
+}
+
+contract SignFromJson is OriginalSignFromJson {
+    using LibString for string;
+
+    // Chains for this task.
+    string constant l1ChainName = "sepolia";
+    string constant l2ChainName = "op";
+
+    // Safe contract for this task.
+    GnosisSafe securityCouncilSafe = GnosisSafe(payable(0xa87675ebb9501C7baE8570a431c108C1577478Fa));
+    GnosisSafe foundationSafe = GnosisSafe(payable(0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B));
+
+    // Contracts we need to check, which are not in the superchain registry
+    IDeputyGuardianModuleFetcher deputyGuardianModule =
+        IDeputyGuardianModuleFetcher(0x2329EfD0bFc72Aa7849D9DFc2e131D83F4680d85);
+    ILivenessGuardFetcher livenessGuard = ILivenessGuardFetcher(0x54E8baCcC67fA3c6b3e9A94BAa4d70d1668f0820);
+    ILivenessModuleFetcher livenessModule = ILivenessModuleFetcher(0xefd77C23A8ACF13E194d30C6DF51F1C43B0f9932);
+
+    // Known EOAs to exclude from safety checks.
+    address constant l2OutputOracleProposer = 0x49277EE36A024120Ee218127354c4a3591dc90A9; // cast call $L2OO "PROPOSER()(address)"
+    address constant l2OutputOracleChallenger = 0xfd1D2e729aE8eEe2E146c033bf4400fE75284301; // In registry addresses.
+    address constant systemConfigOwner = 0xfd1D2e729aE8eEe2E146c033bf4400fE75284301; // In registry addresses.
+    address constant batchSenderAddress = 0x8F23BB38F531600e5d8FDDaAEC41F13FaB46E98c; // In registry genesis-system-configs
+    address constant p2pSequencerAddress = 0x57CACBB0d30b01eb2462e5dC940c161aff3230D3; // cast call $SystemConfig "unsafeBlockSigner()(address)"
+    address constant batchInboxAddress = 0xff00000000000000000000000000000011155420; // In registry yaml.
+
+    // The deployer address which should be removed as an owner on the security council safe.
+    address constant deployerAddress = 0x78339d822c23D943E4a2d4c3DD5408F66e6D662D;
+
+    // Hardcoded data that should not change after execution.
+    uint256 l2GenesisBlockGasLimit = 30e6;
+    uint256 xdmSenderSlotNumber = 204; // Verify against https://github.com/ethereum-optimism/optimism/blob/e2307008d8bc3f125f97814243cc72e8b47c117e/packages/contracts-bedrock/snapshots/storageLayout/L1CrossDomainMessenger.json#L93-L99
+
+    // Data that should not change after execution, fetching during `setUp`.
+    uint256 gasPriceOracleOverhead;
+    uint256 gasPriceOracleScalar;
+    address superchainConfigGuardian;
+    uint256 l2BlockTime;
+    uint256 l2OutputOracleSubmissionInterval;
+    uint256 finalizationPeriodSeconds;
+    uint256 l2OutputOracleStartingTimestamp;
+    uint256 l2OutputOracleStartingBlockNumber;
+    address protocolVersionsOwner;
+    uint256 requiredProtocolVersion;
+    uint256 recommendedProtocolVersion;
+
+    // Other data we use.
+    uint256 systemConfigStartBlock = 4071248; // This was an input when generating input.json
```

Comment:
This is for the sepolia op stack right? 

Based off my understanding of this number, it should be `5116614` as per `Initialized` event at: https://sepolia.etherscan.io/address/0x034edD2A225f7f429A63E0f1D2084B9E0A93b538#events 

I could be wrong here. How did you get `4071248`?

Reply:
As used below in this file it should match the `startBlock()` return value: 

https://github.com/ethereum-optimism/superchain-ops/blob/ff3f8c9733b181c1fc32c9460b82d78f53b08d73/tasks/sep/006-0-sc-guardian-updates/SignFromJson.s.sol#L180

I got it thusly:

```
cast call 0x034edD2A225f7f429A63E0f1D2084B9E0A93b538 'startBlock()(uint)'
4071248
```